packages/-ember-data/tests/integration/identifiers/configuration-test.ts

```diff
@@ -533,7 +533,7 @@ if (IDENTIFIERS) {
             name: 'callback-hell',
           },
           relationships: {
-            retainer: { type: 'retainer', id: '1' },
+            retainer: { data: { type: 'retainer', id: '1' } },
```

Comment:
Can you help me understand this test change? Doesn't immediately seem related to the other changes in this PR to me...

Reply:
once we correctly configured TS and fixed the type issues for relationships on payloads which that uncovered TS alerted us that this test was using a malformed payload.